src/platform/unix/mod.rs

```diff
@@ -72,6 +68,8 @@ lazy_static! {
     };
 }
 
+static GLOBAL_SHM_COUNT: AtomicUsize = ATOMIC_USIZE_INIT;
```

Comment:
Personally, I don't think `PID` should have a `SHM_` prefix -- while its (currently) used only for this, the PID itself is a general property of the process, not in any way SHM-specific...

I don't feel strongly about it though, and I don't think this warrants another iteration :-)